pkg/machines/components/nicEdit.css

```
@@ -1,11 +1,6 @@
 /* Standard dialog is too narrow and wide dialog is too wide */
 .nic-edit > .modal-dialog {
-    width: 70rem;
-}
-
-/* Form has double padding; remove it, except for the last in row */
-.nic-edit .form-group > :not(:last-child) {
-    padding-right: 0;
+    width: 60rem;
```

```suggestion
    max-width: 60rem;
```

Reasoning: We really just want to limit the maximum width, not fix the width to the size.

If the window is sized to be smaller or is on mobile, the dialog might need to be even narrower. If the width is fixed to be exactly 60rem, it will cause overflow issues with scrolling.

---

This would fix an existing bug. (I noticed this issue in my demo while implementing the dialog.)

---

Fixed.